Kernel/Graphics/VMWare/Console.cpp

```diff
@@ -39,7 +39,7 @@ void VMWareFramebufferConsole::flush(size_t, size_t, size_t, size_t)
 
 void VMWareFramebufferConsole::enqueue_refresh_timer()
 {
-    NonnullLockRefPtr<Timer> refresh_timer = adopt_lock_ref(*new (nothrow) Timer());
+    auto refresh_timer = adopt_ref(*new (nothrow) Timer());
```

Comment:
Note for the future: 
This `new` is marked as fallible, but does not use the `nullptr`/failure aware RefPtr API, instead it de-references directly
(which should be a compiler warning to be honest)
The `Console` uses the infallible `new` instead

fallible as in "can return null" instead of "tries to throw an exception/crash on OOM"

Reply:
True, there are a ton of these still around.